AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/AWSDataStorePlugin.swift

```diff
@@ -157,7 +157,7 @@ final public class AWSDataStorePlugin: DataStoreCategoryPlugin {
         case .failure(let dataStoreError):
             dataStorePublisher.send(dataStoreError: dataStoreError)
         case .finished:
-            dataStorePublisher.sendFinished()
+            break
```

Comment:
when/where causes this which we then call `dataStorePublisher.sendFinished()`? and then why change it to not do this?

Reply:
As discussed, the datastore publisher was being finished twice, once in the callback of storageEngine.stopSync/storageEngine.clear and once in this OnReceviedCompletion block. There isn't a need to do this twice. By doing nothing to the `dataStorePublisher` at this point in the code, there is no impact to the behavior of DataStore.clear or stop, so we have decided to remove it